### PR TITLE
Do some stuff to the window that opens when you doubleclick

### PR DESCRIPTION
**Pull request by Paddyk45**
## Type of change

- [x] Bug fix
- [x] New feature

## Description

- Update the wiki link
- Add a "Join our Discord" text and button
- Fix Linux mods path I guess

## Related issues

Mention any issues that this pr relates to.

# How Has This Been Tested?

Tested on Mac and Linux (RPi 4B 8G Raspbian)

# Checklist:

- [x] My code follows the style guidelines of this project.
- [ ] I have added comments to my code in more complex areas.
- [x] I have tested the code in both development and production environments.

### CODE REVIEW

File: launch/src/main/java/meteordevelopment/meteorclient/Main.java [hunk 1]
```diff
@@ -6,36 +6,37 @@
...
 import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.Locale;
+import java.util.concurrent.atomic.AtomicReference;
 
 public class Main {
-    public static void main(String[] args) throws UnsupportedLookAndFeelException, ClassNotFoundException, InstantiationException, IllegalAccessException {
-        UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
```
Comment (arlomcwalter) [inline launch/src/main/java/meteordevelopment/meteorclient/Main.java:17]: why did you remove this?

Comment (Paddyk45) [reply launch/src/main/java/meteordevelopment/meteorclient/Main.java:17]: Oh shit I removed too much

Comment (Paddyk45) [reply launch/src/main/java/meteordevelopment/meteorclient/Main.java:17]: Ok should be fixed

Comment (Paddyk45) [reply launch/src/main/java/meteordevelopment/meteorclient/Main.java:17]: I didn't see that my commit was rejected, let me fix
